Add `4 Digit World Cup 2024` article (#11144)
ppy/osu-wiki: wiki/Tournaments/4WC/2024/en.md

---
tags:
  - 4WC2024
  - 4WC
---

# 4 Digit World Cup 2024

![4WC banner](img/banner.jpg)

The **4 Digit World Cup 2024** (***4WC2024***) is a 4v4, double-elimination, country-based osu! tournament hosted by ::{ flag=DE }:: [resoa](https://osu.ppy.sh/users/6754508) and ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899). It is the sixth instalment of the 4 Digit World Cup.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2024-03-21/2024-04-13 (23:59 UTC) |
| Team submission phase | 2024-04-14/2024-04-20 (23:59 UTC) |
| Qualifier showcase | 2024-04-28 |
| Qualifier stage | 2024-05-04/2024-05-05 |
| Round of 32 | 2024-05-11/2024-05-12 |
| Round of 16 | 2024-05-18/2024-05-19 |
| Quarterfinals | 2024-05-25/2024-05-26 |
| Semifinals | 2024-06-01/2024-06-02 |
| Finals | 2024-06-08/2024-06-09 |
| Grand Finals | 2024-06-15/2024-06-16 |

## Prizes

The 4 Digit World Cup 2024 awards winners with a [community-funded prize pool on Open Collective](https://opencollective.com/4wc).

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Wooting UwU (non-RGB) keyboard for each player, Bluebunny tablet cover, 40% of the prize pool, profile badge (pending), animated profile banner |

Reviewer: not sure if it sounds too clunky adding "for each player" on the tablet covers as well, but should probably not just have it on one of these prizes

Author: just forgot for some reason

```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Wooting UwU (non-RGB) keyboard and Bluebunny tablet cover for each player, 40% of the prize pool, profile badge (pending), animated profile banner |
```

| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 40% of the prize pool, animated profile banner |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 20% of the prize pool, animated profile banner |

## Organisation

The 4 Digit World Cup 2024 is run by various community members.

| Position | Member(s) |
| :-- | :-- |
| Host | ::{ flag=DE }:: [resoa](https://osu.ppy.sh/users/6754508), ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899) |
| Designer and developer | ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899) |
| Mappool selector | ::{ flag=CA }:: [chiv](https://osu.ppy.sh/users/6701656), ::{ flag=DE }:: [resoa](https://osu.ppy.sh/users/6754508) |

> | Mappool playtesters | ::{ flag=RU }:: [_wonderer](https://osu.ppy.sh/users/8215374), ::{ flag=DE }:: [-semi](https://osu.ppy.sh/users/5154946), ::{ flag=PL }:: [cezgru](https://osu.ppy.sh/users/2629617), ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251), ::{ flag=RU }:: [Endura](https://osu.ppy.sh/users/7774197), ::{ flag=PH }:: [enri](https://osu.ppy.sh/users/8640970), ::{ flag=CN }:: [FcEazy](https://osu.ppy.sh/users/7825227), ::{ flag=US }:: [Flameztear](https://osu.ppy.sh/users/13207763), ::{ flag=US }:: [Jakson](https://osu.ppy.sh/users/8788058), ::{ flag=PL }:: [kiir0chii](https://osu.ppy.sh/users/9322480), ::{ flag=US }:: [M I L E S](https://osu.ppy.sh/users/6036351), ::{ flag=GB }:: [Malteser](https://osu.ppy.sh/users/5218178), ::{ flag=ES }:: [MeiFTW](https://osu.ppy.sh/users/15049805), ::{ flag=RO }:: [nanoya](https://osu.ppy.sh/users/12366071), ::{ flag=FI }:: [Nev-](https://osu.ppy.sh/users/11836334), ::{ flag=NO }:: [ninerik](https://osu.ppy.sh/users/10549880), ::{ flag=MN }:: [seegii](https://osu.ppy.sh/users/4659319), ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899), ::{ flag=KR }:: [worst hr player](https://osu.ppy.sh/users/14106450), ::{ flag=DE }:: [ZeitFrost](https://osu.ppy.sh/users/11234706) |

Reviewer: ```suggestion
| Mappool playtester | ::{ flag=RU }:: [_wonderer](https://osu.ppy.sh/users/8215374), ::{ flag=DE }:: [-semi](https://osu.ppy.sh/users/5154946), ::{ flag=PL }:: [cezgru](https://osu.ppy.sh/users/2629617), ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251), ::{ flag=RU }:: [Endura](https://osu.ppy.sh/users/7774197), ::{ flag=PH }:: [enri](https://osu.ppy.sh/users/8640970), ::{ flag=CN }:: [FcEazy](https://osu.ppy.sh/users/7825227), ::{ flag=US }:: [Flameztear](https://osu.ppy.sh/users/13207763), ::{ flag=US }:: [Jakson](https://osu.ppy.sh/users/8788058), ::{ flag=PL }:: [kiir0chii](https://osu.ppy.sh/users/9322480), ::{ flag=US }:: [M I L E S](https://osu.ppy.sh/users/6036351), ::{ flag=GB }:: [Malteser](https://osu.ppy.sh/users/5218178), ::{ flag=ES }:: [MeiFTW](https://osu.ppy.sh/users/15049805), ::{ flag=RO }:: [nanoya](https://osu.ppy.sh/users/12366071), ::{ flag=FI }:: [Nev-](https://osu.ppy.sh/users/11836334), ::{ flag=NO }:: [ninerik](https://osu.ppy.sh/users/10549880), ::{ flag=MN }:: [seegii](https://osu.ppy.sh/users/4659319), ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899), ::{ flag=KR }:: [worst hr player](https://osu.ppy.sh/users/14106450), ::{ flag=DE }:: [ZeitFrost](https://osu.ppy.sh/users/11234706) |
```

non-plural role name

> | Streamer | ::{ flag=US }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=VN }:: [Hoaq](https://osu.ppy.sh/users/7696512), ::{ flag=GB }:: [ilw8](https://osu.ppy.sh/users/14167692), ::{ flag=CA }:: [kymotsujason](https://osu.ppy.sh/users/2541804), ::{ flag=US }:: [ROB_](https://osu.ppy.sh/users/12455868), ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899), ::{ flag=VN }:: [SIay](https://osu.ppy.sh/users/9587896), ::{ flag=US }:: [this1neguy](https://osu.ppy.sh/users/1797189) |

Reviewer 2: ```suggestion
| Streamer | ::{ flag=CA }:: [D I O](https://osu.ppy.sh/users/3958619), ::{ flag=VN }:: [Hoaq](https://osu.ppy.sh/users/7696512), ::{ flag=GB }:: [ilw8](https://osu.ppy.sh/users/14167692), ::{ flag=CA }:: [kymotsujason](https://osu.ppy.sh/users/2541804), ::{ flag=US }:: [ROB_](https://osu.ppy.sh/users/12455868), ::{ flag=FI }:: [shdewz](https://osu.ppy.sh/users/10000899), ::{ flag=VN }:: [SIay](https://osu.ppy.sh/users/9587896), ::{ flag=US }:: [this1neguy](https://osu.ppy.sh/users/1797189) |
```

Reviewer: good catch, still had 2023 data on my staff list it seems

## Links

- **[Registration form](https://osu.ppy.sh/oauth/authorize?client_id=21966&redirect_uri=https://script.google.com/macros/s/AKfycbzknySoKogThpPyGuOGOsmsr1hdCITHOQqF3UXQc-orw-9b_BuEjM129uVUBHmNGWBMpg/exec&response_type=code&scope=identify&state=%7B%22step%22%3A%22osu%22%7D)**
- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1898593)
- [Livestream](https://twitch.tv/4wcb)

> - [Statistics sheet](https://docs.google.com/spreadsheets/d/1gPRSI1lTW8ZMlu_1mnxpVcZyzO2Atbb_2H5nfDt0CXM)

Reviewer: ```suggestion
- [Information spreadsheet](https://docs.google.com/spreadsheets/d/1gPRSI1lTW8ZMlu_1mnxpVcZyzO2Atbb_2H5nfDt0CXM)
```

more fitting in my opinion, and same convention as official world cups over the past year or so

- [Challonge bracket](https://challonge.com/4wc24)
- [Discord server](https://discord.com/invite/uQarcRtB3u)

## Ruleset

### General

- This is a **country-based**, **world-cup-style** tournament.

> - This is a **4v4**, **osu!** tournament with **teams of 5—8 players**.

Reviewer: ```suggestion
- This is a **4v4**, **osu!** tournament with **teams of 5–8 players**.
```

em -> en

- The Qualifiers mappool will consist of 5 No Mod, 2 Hidden, 2 Hard Rock and 3 Double Time picks.
- The **top 32** teams from Qualifiers will advance to a **double-elimination bracket**.
- Grand Finals will feature a **bracket reset** if the team coming from the losers' bracket wins the first match.
- Players outside the rank range are allowed to register. However, if they are not strictly within the rank range by the time registrations close, their registration will be removed.
- After registrations close, players are allowed to leave the rank range.
- Match schedules and other events will be displayed in **UTC time**.
- Matches can be rescheduled if both teams agree to a time and communicate it to the tournament organisers **before Thursday 23:59 UTC** of the week the match is to take place on.
- Staff members, with the exception of streamers and commentators, are not allowed to play in the tournament.
- The tournament organisers reserve the right to modify the rules at any moment.
  - Any changes to the rules will be clearly communicated to the players.

### Conduct

> - All spaces affiliated with 4WC are subject to the [osu! community rules](/wiki/Rules), as well as the rules of the platform itself.

Reviewer: ```suggestion
- All spaces affiliated with 4WC are subject to the [osu! community rules](/wiki/Rules), as well as the rules of the respective platforms themselves.
```

maybe this wording as discussed

- If a country has more than 8 eligible players, they are **recommended to organise tryouts**. If a team has 8 or fewer signups their team will be locked as is. They are, however, free to change their captain or remove a player within the team submission phase.
  - Teams will be provided a pre-made mappool for tryouts that they may choose to use.
- All members of a team **must be registered for the tournament** and have **joined the [Discord server](https://discord.com/invite/uQarcRtB3u)**.
- Teams will range from **5 to 8 members**.
- The captain is allowed to not put themselves on the team if they think it is right to do so, but they are still responsible for submitting the roster to the tournament organisers as well as designating someone else on the roster as the new captain.
- Teams must be submitted **by the end of the team submission phase** (April 20, 23:59 UTC).

### Match format

| Stage | Format |
| :-- | :-- |
| Round of 32 | Best of 9, 1 ban |
| Round of 16 | Best of 9, 1 ban |
| Quarterfinals | Best of 11, 2 bans |
| Semifinals | Best of 11, 2 bans |
| Finals | Best of 13, 2 bans |

> | Grand Finals | Best of 132, bans |

Reviewer: ```suggestion
| Grand Finals | Best of 13, 2 bans |
```
